### PR TITLE
improvement: tekton summer ospp

### PR DESCRIPTION
**Pull request by shaowenchen**
Signed-off-by: shaowenchen <mail@chenshaowen.com>

/assign @kubesphere/sig-devops

### CODE REVIEW

File: sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md [hunk 1]
```diff
@@ -1,56 +1,61 @@
...
 * http://jenkins.io/
-* https://github.com/jenkins-x
-* https://github.com/kubesphere
+* https://github.com/kubesphere/kubesphere
 
 ## 快速入门
 
-You can start with [a minimal KubeSphere](https://kubesphere.io/docs/quick-start/minimal-kubesphere-on-k8s/). [Enable the DevOps](https://kubesphere.io/docs/pluggable-components/devops/) component once KubeSphere is ready and explore KubeSphere DevOps system by yourself.
-[Install Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/install.md) first, then get familiar with various features by following [the tutorial](https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md)!
+你可以从[一个最小化的 KubeSphere](https://kubesphere.io/docs/quick-start/minimal-kubesphere-on-k8s/)开始。KubeSphere 准备好了之后，[启用 DevOps ](https://kubesphere.io/docs/pluggable-components/devops/)组件，并自己探索 KubeSphere DevOps。
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md:48]: This doc is for Chinese students. So using [a Chinese doc](https://kubesphere.io/zh/docs/quick-start/minimal-kubesphere-on-k8s/) will be better.

File: sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md [hunk 2]
```diff
@@ -1,56 +1,61 @@
...
+可以先完成 Tekton 的部署，然后完成 Trigger、Task、Pipeline 的管理，接着是图形化的编辑、运行、查看。在图形化阶段，可以尝试考虑兼容 CRD。
 
 ## 项目产出
 
-The potential outcomes might include:
-* Pipeline CRD compatible with Jenkins
-* Log, and the archive storage solution
-* Multi-branch pipeline support
-* A how-to document that contains the integration, usage, users case
+潜在的结果可能包括
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md:32]: `成果`感觉比`结果`更贴切。

File: sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md [hunk 3]
```diff
@@ -1,56 +1,61 @@
...
+Tekton 是一个云原生的 CI/CD 项目。相较于 Jenkins，Tekton 可以充分利用 Kubernetes 生态系统的优势。例如，Tekton 一个是无服务器的组件，非常容易扩展。然而，由于Tekton 缺乏用户友好的用户界面，新用户可能很难操作它。我们希望能充分利用 Tekton 的优势，使其在 KubeSphere中易于使用。
 
 ## 项目详情
 
-Currently, Jenkins is the engine of KubeSphere DevOps component. The [Pipeline controller](https://github.com/kubesphere/kubesphere/blob/master/pkg/controller/pipeline/pipeline_controller.go) is responsible for converting the CRD of pipelines to Jenkins jobs.
-Please consider the compatibility of the [CRD](https://github.com/kubesphere/kubesphere/blob/master/pkg/apis/devops/v1alpha3/pipeline_types.go). Jenkins remains to be the first choice until we have sufficient reasons to change it.
+目前，Jenkins 是 KubeSphere DevOps 组件的引擎。[Pipeline controller](https://github.com/kubesphere/kubesphere/blob/master/pkg/controller/pipeline/pipeline_controller.go) 负责将管道的CRD 转换为 Jenkins Job。
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md:23]: 社区对 Pipeline 的翻译为`流水线`

另外，下面的语句没有在英文两边保留一个空格：

> 负责将管道的CRD 转换为

File: sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md [hunk 4]
```diff
@@ -1,56 +1,61 @@
...
-* Jenkins
 * Kubernetes
+* Tekton
+* JavaScript[可选]
+* Jenkins[可选]
 
 ## 项目背景
 
-Tekton is a cloud-native CI/CD project. Compared to Jenkins, Tekton can take full advantage of the Kubernetes ecosystem. For example, Tekton, as a serverless component, is very easy to scale. However, it could be difficult for new users to operate Tekton for its lack of user-friendly UI. We would like to make full use of the advantages of Tekton, and make it easy to use in KubeSphere.
+Tekton 是一个云原生的 CI/CD 项目。相较于 Jenkins，Tekton 可以充分利用 Kubernetes 生态系统的优势。例如，Tekton 一个是无服务器的组件，非常容易扩展。然而，由于Tekton 缺乏用户友好的用户界面，新用户可能很难操作它。我们希望能充分利用 Tekton 的优势，使其在 KubeSphere中易于使用。
```
Comment (LinuxSuRen) [inline sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md:19]: > 于Tekton 缺乏用

英文前后保留一个空格

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Thanks for providing the translation and refining. I left some comments below, please consider changing them. Or let's have a discussion if you have different ideas.

File: sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md (hunk 4 above)
Comment (shaowenchen) [reply sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md]: Updated

File: sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md (hunk 2 above)
Comment (shaowenchen) [reply sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md]: Updated

File: sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md (hunk 1 above)
Comment (shaowenchen) [reply sig-advocacy-and-outreach/summer-ospp/kubeSphere-tekton-integration_zh-CN.md]: Updated

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Pull request merged**